index.html

```diff
@@ -0,0 +1,163 @@
+               <p><strong>Cost to Adopt: </strong> $123.45</p>
+               <p>Corrum volorit iandae<br>
+                  nimaxim cum restia volor<br>
+                  reicid ut et etur sunt arum<br>
+                  rendae pla endis re ea erum,<br>
+                  qui doluptae
+               </p>
+               <button onclick="adoptButton('123.45')">Adopt</button>
+            </div>
+            <div class="grid-item">
+               <img src="images/lucas-card.jpg" alt="Lucas Dog" style="width:100%" onclick="viewDogDetails('Lucas','Foo5','123.45')"/>
+               <h4> Lucas </h4>
+               <p><strong>Cost to Adopt: </strong> $123.45</p>
+               <p>Corrum volorit iandae<br>
+                  nimaxim cum restia volor<br>
+                  reicid ut et etur sunt arum<br>
+                  rendae pla endis re ea erum,<br>
+                  qui doluptae
+               </p>
+               <button onclick="adoptButton('123.45')">Adopt</button>
+            </div>
+            <div class="grid-item">
+               <img src="images/jake-card.jpg" alt="Jake Dog" style="width:100%" onclick="viewDogDetails('Jake','Foo6','123.45')"/>
+               <h4> Jake </h4>
+               <p><strong>Cost to Adopt: </strong> $123.45</p>
+               <p>Corrum volorit iandae<br>
+                  nimaxim cum restia volor<br>
+                  reicid ut et etur sunt arum<br>
+                  rendae pla endis re ea erum,<br>
+                  qui doluptae
+               </p>
+               <button onclick="adoptButton('123.45')">Adopt</button>
+            </div>
+            <div class="grid-item">
+               <img src="images/angus-card.jpg" alt="Angus Dog" style="width:100%" onclick="viewDogDetails('Angus','Foo7','123.45')"/>
+               <h4> Angus </h4>
+               <p><strong>Cost to Adopt: </strong> $123.45</p>
+               <p>Corrum volorit iandae<br>
+                  nimaxim cum restia volor<br>
+                  reicid ut et etur sunt arum<br>
+                  rendae pla endis re ea erum,<br>
+                  qui doluptae
+               </p>
+               <button onclick="adoptButton('123.45')">Adopt</button>
+            </div>
+            <div class="grid-item">
+               <img src="images/violet-card.jpg" alt="Violet Dog" style="width:100%" onclick="viewDogDetails('Violet','Foo8','123.45')"/>
+               <h4> Violet </h4>
+               <p><strong>Cost to Adopt: </strong> $123.45</p>
+               <p>Corrum volorit iandae<br>
+                  nimaxim cum restia volor<br>
+                  reicid ut et etur sunt arum<br>
+                  rendae pla endis re ea erum,<br>
+                  qui doluptae
+               </p>
+               <button onclick="adoptButton('123.45')">Adopt</button>
+            </div>
+         </div>
+         <button class="see-more-button"> See More Dogs</button>
+      </main>
+      <footer>
+         <div class="footer">
```

Comment:
A lot of the text on your site is quite large on desktop/mobile, would suggest staying around 1-1.2rem for body text, and only really go above that for headers, titles, etc.